packages/blockchain-wallet-v4/src/signer/bch.js

```diff
@@ -25,10 +25,16 @@ export const signSelection = curry((network, selection) => {
       coin.value
     )
   const sign = (coin, i) => tx.sign(i, coin.priv, null, hashType, coin.value)
+  tx.addInput(
+    coinDust.txHash,
+    coinDust.index,
+    BitcoinCash.Transaction.DEFAULT_SEQUENCE
```

Comment:
Do I need DEFAULT_SEQUENCE?

Reply:
It will be set to that default value if you don't set it:

Reply:
https://github.com/bitcoinjs/bitcoinjs-lib/blob/d78bf6a2b979faa31afdf0a813fc3717b237c28f/src/transaction.js#L164. This is not used by us at the moment, but can be used for RBF: https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki.